websocket-ballerina/tests/29_ssl_echo.bal

```diff
@@ -20,59 +20,62 @@ import ballerina/http;
 
 string expectedString = "";
 byte[] expectedBinaryData = [];
+string expectedRawpath = "";
 
-service UpgradeService /sslEcho on new Listener(21029, {
-       secureSocket: {
-           keyStore: {
-               path: "tests/certsAndKeys/ballerinaKeystore.p12",
-               password: "ballerina"
-           }
-       }
-   }) {
-   remote isolated function onUpgrade(http:Caller caller, http:Request req) returns Service {
+listener Listener l7 = checkpanic new(21029, {
```

Comment:
We don't need checkpanic here ? I see, it seems like the spec has changed. https://github.com/ballerina-platform/ballerina-spec/issues/671

If we need it can't we use check instead of checkpanic ?